<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSet</class>
 <widget class="QDialog" name="DialogSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushConfirm">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushCancel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>390</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>基本设置</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>天</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>秒</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>提醒多久后自动退出（0为不退出）：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAheadDays">
    <property name="geometry">
     <rect>
      <x>237</x>
      <y>170</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxAutoRun">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>开机自动启动</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRemindTime">
    <property name="geometry">
     <rect>
      <x>237</x>
      <y>110</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxCloseMin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭窗口时最小化到系统托盘</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>秒</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxStartMin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>启动时最小化到系统托盘</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>程序运行多久后提醒：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditQuitTime">
    <property name="geometry">
     <rect>
      <x>237</x>
      <y>140</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>提前多少天开始提醒：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>381</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>导入/导出设置</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>数据格式：姓名|出生日期|生日类型
如：张三|2000-1-1|公历</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>数据分隔符（默认'|'）：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDataSep">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>|</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>日期分隔符（默认'-'）：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTimeSep">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
